third_party/terraform/tests/resource_dataflow_job_test.go

```diff
@@ -182,6 +183,35 @@ func TestAccDataflowJob_withIpConfig(t *testing.T) {
 	})
 }
 
+func TestAccDataflowJob_streamUpdate(t *testing.T) {
```

Comment:
Oh, how possible would it be to do one that tests the contents of the template changing?

Reply:
Should be possible, and a good test to add. I'll cook one up.